packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -87,6 +93,27 @@ const HeaderTools = ({
                 document.querySelector('html').classList.toggle('pf-v5-theme-dark')} />
             </ToolbarItem>
           )}
+          {hasRTLSwitcher && (
+            <ToolbarItem>
+              <ToggleGroup aria-label="Example direction toggle">
+                <ToggleGroupItem
+                  icon={isRtl ? <AlignRightIcon /> : <AlignLeftIcon />}
+                  aria-label="Toggle direction"
+                  buttonId="ws-rtl-toggle"
+                  isSelected={false}
+                  onChange={() => setIsRtl(isRtl => {
+                    const examples = document.querySelectorAll('.ws-example');
+                    examples.forEach(ex => ex.classList.toggle('pf-v5-m-dir-rtl'));
+                    return !isRtl;
+                  })}
+                />
+              </ToggleGroup>
+              <Switch id="ws-rtl-switch" label="Example direction" defaultChecked={false} onChange={() =>{
```

Comment:
I like the switch. to save some real estate we could switch the toggle between "RTL" and "LTR" text.